kubetest2/kubetest2-gce/deployer/deployer.go

```diff
@@ -67,7 +69,32 @@ func (d *deployer) Provider() string {
 	return Name
 }
 
-func (d *deployer) IsUp() (up bool, err error) { return false, nil }
+func (d *deployer) IsUp() (up bool, err error) {
+	klog.Info("GCE deployer starting IsUp()")
+
+	if err := d.verifyFlags(); err != nil {
+		return false, fmt.Errorf("is up failed to verify flags: %s", err)
+	}
+
+	env := d.buildEnv()
+	// naive assumption: nodes reported = cluster up
```

Comment:
interesting, they don't check status?

Reply:
Yep, both the kind and gke deployers do almost exactly this block of code.